packages/components/src/internal/components/domainproperties/constants.ts

```diff
@@ -209,7 +209,8 @@ export const DEFAULT_DOMAIN_FORM_DISPLAY_OPTIONS = {
     hideValidators: false,
     isDragDisabled: false,
     hideTextOptions: false,
-    phiLevelDisabled: false,
+    phiLevelEnabled: true,
+    phiLevelDisabledReason: null,
```

Comment:
I don't believe that these two are needed here anymore. They have been properly removed from the `IDomainFormDisplayOptions` interface and are now being set on the `DomainDesign` model.